compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -351,16 +426,11 @@ void exportOpDefinedTensor(const CircleTensoInfo &info, FlatBufferBuilder &build
   if (info.shape_status() == ShapeStatus::VALID)
     shape_offset = encodeShape(builder, info.shape());
 
-  // encode and register output tensor buffer
-  auto buffer =
-      info.content() == nullptr ? encodeOpBuffer(builder) : encodeOpBuffer(builder, info.content());
-
   auto quantparam = encodeQuantizationParameters(builder, info.quantparam());
 
   auto sparsityparam = encodeSparsityParameters(builder, info.sparsityparam());
 
-  auto buffer_id = static_cast<uint32_t>(md._buffers.size());
-  md._buffers.push_back(buffer);
+  auto buffer_id = get_buffer_id(builder, md, info.content());
```

Comment:
@llFreetimell 
I'm not sure but as original code said, buffer is created even if `info.content()` is `nullptr`. But changed code doesn't. Is this one of what you intended?

Reply:
Yes it was intended.
When `info.content()` is `nullptr`, empty buffer is newly generated every time.
However, it can be substituted by buffer id 0, which is representative empty buffer.

Reply:
> However, it can be substituted by buffer id 0, which is representative empty buffer.

I'm not sure this is OK but I'll go with your decision.